vue-components/stories/Icon.stories.ts

```diff
@@ -0,0 +1,25 @@
+import Icon from '@/components/Icon';
+import { iconSizes, iconColors, iconTypes } from '@/components/iconProps';
+import { Component } from 'vue';
+
+export default {
+	component: Icon,
+	title: 'Icon',
+};
+
+export function all(): Component {
+	return {
+		data(): object {
+			return { iconTypes };
+		},
+		components: { Icon },
+		template: `
+			<div>
+				<div v-for="type in iconTypes" style="float: left; padding: 20px 50px; color: #555">
+					<Icon :type="type" />
```

Comment:
Leaving this here FTR - https://www.npmjs.com/package/@storybook/addon-knobs#color